bootstrap3-typeahead.js

```diff
@@ -98,12 +99,15 @@
         });
         this.$element.after(this.$statusRegion);
 
-        this.$element.attr({
+        this.$parent.attr({
             'role': 'combobox',
-            'aria-autocomplete': 'both',
             'aria-haspopup': 'listbox'
         });
 
+        this.$element.attr({
+            'aria-autocomplete': 'both'
```

Comment:
I know this isn't your fault but I think this should actually be `list` as per the example because `both` implies we also do inline autocomplete inside the text field, which we totally don't

Reply:
Done'd